src/QsCompiler/Compiler/PluginInterface.cs

```diff
@@ -9,6 +9,18 @@
 
 namespace Microsoft.Quantum.QsCompiler
 {
+    /// <summary>
+    /// Lists the priorities for built-in rewrite steps. 
+    /// </summary>
+    public static class RewriteStepPriorities
+    {
+        public const int ControlFlowSubstitutions = 1100;
```

Comment:
I'd suggest some brief API documentation comments for these.

Reply:
done